scripts/index_generator.py

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim:fenc=utf-8
+#
+
+"""
+
+"""
+import os
+from jinja2 import Environment
+
+index_tpl = """
+<html>
+<title>{% block title %}Open Programmable Acceleration Engine{% endblock %}
+</title>
+<body bgcolor='#85C1E9'>
+<h1>Open Programmable Acceleration Engine</h1>
+<ul>
+{% for ver in versions %}
+  <li><a href="{{ ver }}/index.html">{{ ver }}</a></li>
+{% endfor %}
+</ul>
+</body>
+</html>"""
+
+
+def main():
+    versions = []
+    for x in os.listdir('.'):
+        if os.path.isdir(x):
```

Comment:
a regex might be better...
```Python
import re
...
for x in os.listdir(somedir):
    if re.match('\d+\.\d+\.\d+', x):
        versions.append(x)
```

This way, we only look at directories that match a version format. This example only works if the versions are numeric and ignores `latest` directory. Maybe something we can do later?

Reply:
As long as someone writes it down for later. :)